### ISSUE TITLE
Oobabooga support?

### ISSUE THREAD

**Issue by Ainaemaet** [iGavroche/BrainChulo#78]
Hello, been following this project for some time now - and it's never been exactly clear to me if it can be used with Oobabooga yet.

The front page description regarding the matter seems outdated and doesn't have much info, so I just wanted to ask: Does BrainChulo work with Oobabooga? And if yes, how to set it up - and if no, is there plans to make it so?

**Comment by iGavroche** [comment]
Hi @Ainaemaet, thanks for your interest in BrainChulo.
We had started the project with Oobabooga support but eventually diverged when we introduced guidance and logprobs; which Ooba did not support at the time. They had also rejected our PR for support, stating there was no impetus to maintain this feature.
In any case, @Karajan421 is working on a next-generation system which should bring open LLMs to a desired level of understanding when it comes to documents and vector-based database knowledge.

Re-enabling Ooba shouldn't be hard but it's a step in a direction we aren't really considering going back to for the moment, unless a kind user submits a PR for it.
